### DISCUSSION TITLE
Use Diffent Claude models

### DISCUSSION THREAD

**Discussion by kmgroen** [cline/cline#695] [Feature Requests]
Sorry guys, this must be somewhere but I couldn't find it. 

I want to use different claude models then by default avaialble in the model drop down. Basically to bypass the daily limit and avoid needing to go to tier 3. E.g. I want to be able to select the older 3.5 Sonnet model. 

How to do that?

**Comment by thebobbyboy420** [comment]
use openrouter api key
